lib/phoenix/live_dashboard/helpers/tree_helpers/reingold_tifford_algorithm.ex

```diff
@@ -0,0 +1,239 @@
+defmodule Phoenix.LiveDashboard.ReingoldTilford do
+  # Reingold-Tilford algorithm for drawing trees
+  @moduledoc false
+
+  @node_height 30
+  @node_y_separation 10
+  @total_y_distance @node_height + @node_y_separation
+  @node_x_separation 50
+
+  def set_layout_settings(tree) do
+    tree
+    |> change_representation(0)
+    |> calculate_initial_y(0, [])
+    |> ensure_children_inside_screen()
+    |> put_final_y_values(0)
+    |> put_x_position(:first_call)
+  end
+
+  defp change_representation({{_, pid, name}, children}, level) do
+    children =
+      Enum.reduce(children, [], fn node, acc ->
+        [change_representation(node, level + 1) | acc]
+      end)
```

Comment:
This can probably be an Enum.map now!

Reply:
Noob question... why?

Reply:
To clarify: Enum.reduce + Enum.reverse is the same as Enum.map. :)

Reply:
Thanks!